src/storages/user.storage.js

```diff
@@ -14,35 +16,63 @@ const createUser = (req, res) => {
     }
 
     // saved!
-    res.status(200).json({ message: "User cerated!", doc });
+    res.status(200).json({ message: "User created!", doc });
   });
 };
 
+//an ADMINISTRATOR can obtain a user from its id
+//togli la password
 const getUser = (req, res) => {
-  // TODO: Return some info about user, not sensitive info like password.
-  console.log("Get user", req.params.id);
-};
-
+  var id = req.params.id;
+  User.findById(id, function (err, docs) {
+    if (err) {
+      console.log(err);
+      return res.status(400).send(err)
+    }
+    else {
+      console.log("Result : ", docs);
+      res.json(docs)
+    }
+  });
+}
+/*
+//these options are for the users-> to do in auth storage
 const updateUser = (req, res) => {
-  // TODO: Update some info about user.
-  console.log("Update user", req.params.id);
+  // TODO: An administrator can update info of a user?
```

Comment:
If we doesn't need anymore some code, you delete it or move to the new location. Leaving pending commented code is a bad practice.

Reply:
tolto